docbook/gnc-locale-pt.dtd

```diff
@@ -2,7 +2,7 @@
      Only parameter entities (% entities) should be translated.
      See htps://www.w3.org/TR/REC-xml/#sec-physical-struct. -->
 
-
+<!ENTITY % euro '€'>
```

Comment:
As we use UTF-8  encoding you  can directly enter `€` [altgr]+{E] on my keyboard.
There is also already
`<!ENTITY euro SDATA "[euro  ]"><!-- euro sign -->`
in the standard docbookx.dtd